.github/workflows/cmake.yml

```diff
@@ -21,13 +21,13 @@ jobs:
       - name: make build dir
         run: cmake -E make_directory ${{runner.workspace}}/build
       - name: configure cmake
-        if: matrix.os != 'windows-latest'
         run: cmake -S ${{ github.workspace }} -B ${{runner.workspace}}/build
-      - name: configure cmake (WIN)
-        if: matrix.os == 'windows-latest'
-        run: cmake -S ${{ github.workspace }} -B ${{runner.workspace}}/build -G "MinGW Makefiles"
```

Comment:
why not MinGW? CGO requires MinGW build IIRC

Reply:
i didn't see the failure locally so i am changing this to be the same as to how i ended up building it from the go script. that uses `-G Ninja` which uses mingw

Also I was just hoping to see it compile since the go project buidls this each time anyway